source/class.cpp

```diff
@@ -1172,12 +1185,24 @@ void ClassBinder::bind(Context &context)
 	string const trampoline_name = callback_structure_constructible ? callback_structure_name(C) : "";
 	string const binding_qualified_name = callback_structure_constructible ? callback_structure_name(C) : qualified_name;
 
+	bool use_custom_shared = Config::get().use_custom_shared();
+	string const custom_shared = Config::get().custom_shared();
+
+	// Do not use custom_shared pointer if std:: class
+	if( qualified_name.rfind("std::", 0) == 0 ) use_custom_shared = false;
```

Comment:
Same as above: why should we exclude `std::` classes from using custom-shared-pointer-class?